README.md

```diff
@@ -51,6 +51,7 @@ zfs | Exposes [ZFS](http://open-zfs.org/) performance statistics. | [Linux](http
 
 Name     | Description | OS
 ---------|-------------|----
+bcache | Exposes bcache statistics | Linux
```

Comment:
It's a bit arbitrary, and some of them probably could move to the default enabled list with a bit of cleanup to handle missing systems gracefully.  Maybe we can make that a goal of 0.15.